docs/source/tools/assistant.rst

```diff
@@ -178,26 +178,32 @@ The generated ``.dar`` file is created in ``.daml/dist/${name}.dar`` by default.
 Managing SDK releases
 *********************
 
-In general the ``daml`` assistant will install versions and guide you when you need to update SDK versions or project settings. If you disable ``auto-install`` and ``update-check`` in the global config file, you will have to manage SDK releases manually.
+You can manage SDK releases manually by using ``daml install``.
 
-To download and install the latest stable SDK release and update the assistant, run::
+To download and install the latest stable SDK release::
 
-  daml install latest --activate
+  daml install latest
 
-Remove the ``--activate`` flag if you only want to install the latest release without updating the ``daml`` assistant in the process. If it is already installed, you can force reinstallation by passing the ``--force`` flag. See ``daml install --help`` for a full list of options.
+To download and install the latest snapshot release::
```

Comment:
Maybe a good idea to add a note that these are not intended for production usage.

Reply:
Good idea, added!

Reply:
Thanks!